serverless/about-serverless.adoc

```diff
@@ -0,0 +1,78 @@
+[id="about-serverless"]
+= About {ServerlessProductName}
+:context: about-serverless
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+{ServerlessProductName} enables developers to deploy and manage cloud native applications that can run anywhere, without provisioning or managing servers.
+These routine tasks are abstracted away by the platform, allowing developers to push code to production much faster than in traditional models.
+
+{ServerlessProductName} is based on the open source link:https://knative.dev/[Knative] project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
+
+Developers can use a single platform for hosting their microservices, legacy, and serverless applications, and can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and stateless container workloads.
+This improves developer productivity by removing the need for complex infrastructure set up before apps can be deployed.
+
+Operational costs can be reduced by implementing out of the box {ServerlessProductName} solutions for automating operational tasks, such as scaling apps, by installing the {ServerlessProductName} Knative Serving and Eventing components.
+You can fine tune system and application settings by creating or modifying Kubernetes custom resources.
+
+include::modules/serverless-supported-configs.adoc[leveloffset=+1]
+
+[id="about-serverless-serving"]
+== Knative Serving component
+
+Knative Serving supports deploying and managing link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native applications] by providing a set of objects as Kubernetes Custom Resource Definitions (CRDs) that define and control the behavior of serverless workloads on an {product-title} cluster.
+For more information about CRDs, see xref:../operators/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[Extending the Kubernetes API with Custom Resource Definitions].
+
+Developers use these CRDs to create custom resource (CR) instances that can be used as building blocks to address complex use cases. For example:
+
+* Rapidly deploying serverless containers.
+* Automatically scaling pods.
+// TODO: Add other use cases, more advanced tutorials?
+
+For more information about CRs, see xref:../operators/crds/crd-managing-resources-from-crds.adoc#crd-managing-resources-from-crds[Managing resources from Custom Resource Definitions].
+
+=== Knative Serving CRDs
+// TODO: Request a good Serving arch diagram
+
+Service:: The `service.serving.knative.dev` CRD automatically manages the life cycle of your workload to ensure that the application is deployed and reachable through the network. It creates a Route, a Configuration, and a new Revision for each change to a user created Service, or custom resource. Most developer interactions in Knative are carried out by modifying Services.
```

Comment:
* Should "Revision" and "Configuration" be lowercase? (Same elsewhere in the PR)
* user created -> user-created?
* Probably lose the comma before custom resource

Reply:
I don't think so re lowercase - It's kind of like service vs. Service, these are referring to a specific type of object as opposed to the general concept of "configurations" or "revisions" etc, so it makes sense to me to give them caps?

This should probably be part of the overall discussions happening around Kubernetes objects, what to capitalize etc, so any guidance is appreciated, I'm sort of waiting to see what the OpenShift guidelines end up being before I recommend anything specific for Serverless or Knative :slightly_smiling_face: 

Maybe a follow up issue/PR to track adding these specific resource types to the glossary would be worthwhile?